content/general/concepts/octal/octal.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'Octal' 
+Description: 'Octal is a number system that uses digits from 0 to 7. Compared to the decimal system each digit corresponds to the power of 8 instead of the power of 10.' 
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Code Foundations'
+Tags: 
+  - 'Arithmetic'
+  - 'Encoding'
+CatalogContent: 
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+The **octal** number system can be compared with the decimal, the hexadecimal, the [binary](https://www.codecademy.com/resources/docs/general/binary), and many other systems. As the name says the system consists of 8 different digits (0-7). In our daily life, we are used to the decimal system, which consists of steps to the power of 10. Instead the Octal system uses steps to the power of 8.
```

Comment:
This sentence should be reworded to include the definition of the octal system.

Reply:
Something like: "The **octal** number system is a number system that is base 8. The traditional number system that we are used to is decimal which is base 10."